docs/networking/rocm.rst

```diff
@@ -0,0 +1,119 @@
+ROCm
+====
+
+How do I build Open MPI with ROCm-aware support?
```

Comment:
Don't feel like this has to be question-answer style like the CUDA help.  Indeed, it would be better if it were more of a declarative style, like the rest of the docs.

I think the CUDA file is largely based on [its corresponding FAQ section](https://www.open-mpi.org/faq/?category=buildcuda), and hasn't (yet) been converted over to a more declarative style.

E.g., you could just name this section "Building Open MPI with ROCm-aware support".

The same goes for the other sections in this page.